components/climber.py

```diff
@@ -0,0 +1,31 @@
+class Climber:
+    SPEED = 1.0
+
+    def __init__(self) -> None:
+        self.will_reset_to = 0.0
```

Comment:
There is a slight misunderstanding here. `target_speed` is the right name for this, but it should be of type `will_reset_to`. Have a look at how the `voltage_set_point` member variable is defined in the `CoralShooterComponent` on main.